raddb/mods-available/always

```diff
@@ -10,8 +10,6 @@
 #
 #  The `always` module simply returns the same result, always, without
 #  doing anything else.  It is here for use in complex policies.
-#
-
```

Comment:
I really don't like deleting these trailing comments.  The example configs use this a *lot*